dogfood/Dockerfile

```diff
@@ -8,7 +8,7 @@ FROM ubuntu:jammy AS go
 
 RUN apt-get update && apt-get install --yes curl gcc
 # Install Go manually, so that we can control the version
-ARG GO_VERSION=1.20
+ARG GO_VERSION=1.20.3
```

Comment:
Tbh I feel like keeping the patch out is fine so it always grabs the latest patch